test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -1107,6 +1107,31 @@ TEST_P(ExtProcIntegrationTest, BufferBodyOverridePostWithEmptyBodyStreamed) {
   verifyDownstreamResponse(*response, 200);
 }
 
+// Test how the filter responds when asked to stream a request body for a POST
+// request with an empty body. We should get an empty body message because
+// the Envoy filter stream received the body after all the headers.
```

Comment:
Should this comment mention that it's using buffered partial?

Reply:
Yes!